docs/design/hierarchical-queue-on-capacity-plugin.md

```diff
@@ -0,0 +1,118 @@
+# Hierarchical Queue on Capacity Plugin
+
+## Motivation
+
+In the context of cloud-native AI task scheduling, fair scheduling and resource utilization are two critical concerns for users. The Volcano community has already developed the elastic queue management plugin, known as capacity, to support fine-grained resource borrowing/lending and queue management, thereby improving resource utilization. However, in real-world scenarios, queues are often hierarchical, corresponding to the hierarchical organizational structure of company teams. To better align with practical queue usage scenarios and further enhance resource utilization for AI task scheduling, Volcano needs to build upon the existing capacity plugin and introduce hierarchical queue management capabilities.
+
+## In Scope
+
+- Enhancing the elastic queue management capabilities of the capacity plugin to support hierarchical queues.
```

Comment:
Will this proposal add a new CR object for users to create? Maybe ChildQueue? Or is this completely internal logic? Will it not be exposed to users?

Reply:
This proposal only involves internal logic changes within the Capacity plugin. It does not introduce any new CR. 

The existing Queue CR already has a parent field. Internally, the Capacity plugin will establish the hierarchical relationships between queues based on the parent field of all the Queues.

Reply:
Will there be any changes to the Queue CR API? If so, we should also show it in the documentation.

Reply:
It does not require any changes to the Queue CR API. From a user's perspective, the way they create and manage Queue CRs will remain the same.